### DISCUSSION TITLE
Skeleton on the Logrocket Podcast

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#970] [Announcements]
![podrocket](https://user-images.githubusercontent.com/1509726/219106935-38553908-46d1-45e4-8327-1681147c1aba.jpg)

Hey everyone, we're excited to share that core maintainer [Chris Simmons](https://github.com/endigo9740) recently appeared on an episode of the [LogRocket podcast](https://podrocket.logrocket.com/) to talk about Skeleton. [LogRocket](https://blog.logrocket.com/) has a huge reach in the web development community so we're very humbled by this opportunity. This comes with a bit of a tease for the upcoming release candidate - so you're not going to want to miss this!

## About the Show

PodRocket covers everything you need to know about frontend web development on a weekly basis. Join LogRocket cofounder Ben Edelstein, the LogRocket engineering team, and more, as they interview experienced developers about all the libraries, frameworks, and tech industry issues they deal with every day.

## Where

https://podrocket.logrocket.com/skeleton

Subscribe and listen from any of these sources:
https://podrocket.logrocket.com/subscribe

**Comment by drmikecrowe** [comment]
Thanks @endigo9740 for doing that podcast.  I'd been curious about Svelte, and that pushed me over to doing a project in it!

**Reply by endigo9740** [reply]
Awesome, glad to hear that Mike! It's a few years old at this point, but this was the video that really convinced me to move over. It's Rich Harris (creator of the framework) talking about the inspiration and mechanics for how it operates compared to alternatives:
https://www.youtube.com/watch?v=AdNJ3fydeao
